frictionless/project/filesystem.py

```diff
@@ -48,7 +48,7 @@ def copy_file(self, path: str, *, folder: Optional[str] = None) -> str:
         return path
 
     # TODO: use streaming?
-    def create_file(
+    def upload_file(
```

Comment:
@shashigharti 
Actually, we don't need to do so we can keep just one `filesystem.create_file` endpoint and use it in both:
- ` project.create_file`
- `project.upload_file`